modules/custom-security-policies-about.adoc

```diff
@@ -14,4 +14,9 @@
 * You can use the {product-title-short} portal to create and save a new policy.
 * With the policy as code feature, you can create and save policies as Kubernetes custom resources (CRs) and use a Kubernetes-native continuous delivery (CD) tool such as Argo CD to apply them to clusters.
 
-For more information, see "Creating and modifying custom security policies".
\ No newline at end of file
+For more information, see "Creating and modifying custom security policies".
+
+[IMPORTANT]
+====
+Use caution when creating custom policies and applying them to `openshift-*` namespaces and default {ocp} pods.
```

Comment:
when enforcing customer policies in openshift-* .....and for default ....

Rationale: Just creating the policies should be ok. Enabling them is also ok. Enforcing them is a problem.